File: abc/abc263/a.py
# https://atcoder.jp/contests/abc263/submissions/34863957

a, b, c, d, e = map(int, input().split())
a, b, c, d, e = sorted([a, b, c, d, e])
if a == b == c != d == e or e == d == c != b == a: print("Yes")
else: print("No")

File: abc/abc263/a.java
// https://atcoder.jp/contests/abc263/submissions/43810483

import java.util.Scanner;
import java.util.Arrays;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int[] a = new int[5];
    for(int i = 0; i < 5; i++) a[i] = stdIn.nextInt();
    Arrays.sort(a);

    int idx = -1;
    for(int i = 1; i < 5; i++) {
      if(a[i - 1] != a[i]) {
        if(idx != -1) {
          System.out.println("No");
          System.exit(0);
        }

        idx = i;
      }
    }

    if(idx == 2 || idx == 3) {
      System.out.println("Yes");
    } else {
      System.out.println("No");
    }
  }
}
